airflow-core/src/airflow/models/taskinstance.py

```diff
@@ -1399,13 +1400,14 @@ def _asset_event_extras_from_aliases() -> dict[tuple[AssetUniqueKey, frozenset],
                 if alias_name not in outlet_alias_names:
                     continue
                 asset_key = AssetUniqueKey(**event["dest_asset_key"])
-                extra_key = frozenset(event["extra"].items())
-                d[asset_key, extra_key].add(alias_name)
+                extra_json = json.dumps(event["extra"], sort_keys=True, ensure_ascii=True)
```

Comment:
```suggestion
                extra_json = json.dumps(event["extra"], sort_keys=True)
```

Probably don’t need to be ASCII?

Reply:
yep, this is not user-facing. should be good